docs/developers/guides/wallet_integration.md

```diff
@@ -135,6 +131,93 @@ let broadcastPost = await fetch(
 let response = await broadcastPost.json();
 ```
 
+#### Sign and Broadcast EVM Transactions
+
+Developers can use Metamask or Keplr to sign off on EVM transactions as well, to manage NFTs, exchange ERC-20 tokens, and more.
```

Comment:
is this the same process for signing with Ledger? if so we should add that to the subtitle